pandas/computation/pytables.py

```diff
@@ -129,7 +129,7 @@ def conform(self, rhs):
         """ inplace conform rhs """
         if not com.is_list_like(rhs):
             rhs = [rhs]
-        if hasattr(self.rhs, 'ravel'):
+        if hasattr(rhs, 'ravel'):
```

Comment:
actually this might be better to call: ``np.array(rhs)``, I don't really remember why I did this in the first place. Maybe you can divine some things from the test cases.